Bottom: 4ff01026dfc04974ff5bc2abfaa2f4863421fc02
Top:    31cd0ed376741ff9a1ce0385398241657d813006
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-07 14:12:15 -0500



---
diff --git a/docker/docker-mantisbt/docker-entrypoint.sh b/docker/docker-mantisbt/docker-entrypoint.sh
index c517f76a..2a948ac8 100755
--- a/docker/docker-mantisbt/docker-entrypoint.sh
+++ b/docker/docker-mantisbt/docker-entrypoint.sh
@@ -109,7 +109,7 @@ case ${1} in
         # Apache gets grumpy about PID files pre-existing
         rm -f /var/run/apache2/apache2.pid
         # Start apache
-        exec apache2 -D FOREGROUND
+        exec apache2-foreground
         ;;
 
     config_inc)
